package.json

```diff
@@ -42,9 +44,9 @@
     "prettier": "^2.0.5",
     "ts-jest": "^26.3.0",
     "ts-node": "^9.0.0",
-    "tslint": "^6.1.3",
-    "tslint-config-prettier": "^1.18.0",
-    "tslint-plugin-prettier": "^2.3.0",
     "typescript": "^4.0.2"
+  },
+  "dependencies": {
+    "react": "^16.13.1"
```

Comment:
Our eslint config has configuration for react code. There is no React in this repo so I had to uninstall to run eslint. Not sure if its ok, probably not. Seems like a topic to discuss in order to handle it correctly.

Reply:
The react dependency should be added in the lib which uses it. This isn't the root lib dependency. However, if you would like to require the dependency from the host project (which use your plugin) you can use the [Peer Dependencies](https://nodejs.org/es/blog/npm/peer-dependencies/).

Reply:
Please let me know if I have answered your question 🙃 .

Reply:
After all those change it was possible to remove React without any problems ced7091e7fadb8c4969786231f81889b70344325